src/components/Pagination/Pagination.test.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import { Pagination, mapDispatchToProps } from './index';
+
+const { mount } = Enzyme;
+
+const defaultProps = {
+  assetsListMetaData: {
+    pageSize: 50,
+    totalCount: 5000,
+  },
+  updatePage: () => {},
+};
+
+const totalPages = Math.floor(
+  defaultProps.assetsListMetaData.totalCount / defaultProps.assetsListMetaData.pageSize,
+);
+
+describe('<Pagination />', () => {
+  let wrapper;
+  let pageItems;
+  let pageLinks;
+
+  beforeEach(() => {
+    wrapper = mount(
+      <Pagination
+        {...defaultProps}
+      />,
+    );
+
+    pageItems = wrapper.find('.page-item');
+    pageLinks = wrapper.find('.page-item .page-link');
+  });
+
+  it('renders correct number of pages', () => {
+    expect(parseInt(pageItems.last().text(), 10)).toEqual(totalPages);
+    expect(parseInt(pageLinks.last().text(), 10)).toEqual(totalPages);
+  });
+  it('page click calls updatePage', () => {
+    const updatePageSpy = jest.fn();
+
+    wrapper.setProps({ updatePage: updatePageSpy });
+
+    pageLinks.last().simulate('click');
+
+    expect(updatePageSpy).toHaveBeenCalledTimes(1);
+    // API treats pages as 0-indexed, but we treat them as 1-indexed
+    expect(updatePageSpy).toHaveBeenCalledWith(totalPages - 1);
+  });
+  it('has correct mapDispatchToProps', () => {
```

Comment:
We have not been testing mapDispatchToProps, but this is a way to test it that I found recommended online, so it might be useful to add this sort of test to the other components (at a later date).